apps/webapp/src/components/send/ibc/ibc-form.tsx

```diff
@@ -1,33 +1,15 @@
 import { Button } from '@penumbra-zone/ui';
-
 import { useToast } from '@penumbra-zone/ui/components/ui/use-toast.ts';
 import { useStore } from '../../../state';
 import { ibcSelector } from '../../../state/ibc.ts';
-import InputToken from '../../shared/input-token.tsx';
-import { amountToBig } from '../../../state/send.ts';
-import { useLoaderData } from 'react-router-dom';
-import { AssetBalance } from '../../../fetchers/balances.ts';
-import { uint8ArrayToBase64 } from '@penumbra-zone/types';
-import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1alpha1/num_pb';
 import { ChainSelector } from './chain-selector.tsx';
 import { InputBlock } from '../../shared/input-block.tsx';
 
+// TODO: Re-implementing ibc form, see send form
```

Comment:
Could you follow up with a second PR to implement this for IBC as well? (as well as the validations plz)